inc/saimacsec.h

```diff
@@ -825,6 +827,7 @@ typedef enum _sai_macsec_sa_attr_t
 
     /**
      * @brief SSCI value for this Secure Association
+     * Host Byte Order.
```

Comment:
Hi  @Pterosaur 
Should this be in Network byte order?

Reply:
The current implementation we used is host byte order. Meanwhile,  some vendors had implemented their SAI according to Host Byte Order.
In my opinion, SSCI will be encoded into frame, So, I think to use Host Byte Order looks reasonable too.